test_scripts/Resumption/InteriorVehicleData/023_InteriorVD_repeated_subscription.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
```

Comment:
@GetmanetsIrina Please add link to proposal as in other scripts for this feature